dgraph/cmd/zero/zero_test.go

```diff
@@ -112,3 +118,87 @@ func TestProposalKey(t *testing.T) {
 	}
 	require.Equal(t, len(uniqueKeys), 10, "each iteration should create unique key")
 }
+
+func validateJsonHealthContent(body []byte) error {
+	var r map[string]interface{}
+	e := json.Unmarshal(body, &r)
+	if e != nil {
+		return errors.Wrap(e, "json unmarshal error")
+	}
+
+	for k, v := range r {
+		switch k {
+		case "instance", "address", "status", "version":
+			s, e := v.(string)
```

Comment:
it would be okay to skip the error here because this is a test and if type is not string it'd panic which is kinda what we want.